app/models/git/concept.rb

```diff
@@ -0,0 +1,36 @@
+module Git
+  class Concept
+    extend Mandate::Memoize
+    extend Mandate::InitializerInjector
+
+    initialize_with :track_slug, :concept_slug, :git_sha
+
+    memoize
+    def data
+      resp = RestClient.get(url_for(:data))
+      data = JSON.parse(resp.body)
+      OpenStruct.new(data['concept']).tap do |concept|
```

Comment:
Jeremy might not agree (I object). But making these named ValueObject classes would have my preference.

Reply:
Yeah - I'd prefer that too normally. 

However, as I don't know if I'm going to roll all this back into the website rather than having a seperate service at all, quick and dirty works for me for now.